packages/runner-ct/lib/runner-ct.js

```diff
@@ -56,7 +56,14 @@ module.exports = {
 
     return res.render(runnerPath, {
       base64Config,
+      namespace: config.namespace,
       projectName: config.projectName,
     })
   },
+  serveChunk (req, res, options) {
+    let { config } = options
+    let pathToFile = getPathToDist(req.originalUrl.replace(config.clientRoute, ''))
```

Comment:
Why is there a pathToDist at all? We do not write out to the file system.

Reply:
This executes in the nodejs

Reply:
I think this is for serving a chunk such as `event-manager` as done above

Reply:
Yeah it was unclear to me what the context of this was.. if it was for userspace or for us to serve runner-ct in test mode. I understand now.